xml/rmt_rmt-cli.xml

```diff
@@ -314,7 +313,7 @@
     <term><command>rmt-cli systems scc-sync</command></term>
     <listitem>
      <para>
-      This command forwards registered systems data to SCC.
+      This command forwards registered systems' data to SCC.
```

Comment:
I think we'd usually do `systems's`...

Reply:
"Systems's" is **definitely** incorrect, and what I made was the recommendation from the proofreading team in Dublin, which is good, correct English.

Reply:
You are right. The rule is that both `s'` and `s's` work for possessives of singulars (with Brits preferring `s'` and Americans split on the issue), and always `s'` when there's a plural possessive, such as this one.